apps/confidential/confidential.go

```diff
@@ -92,16 +91,25 @@ func CertFromPEM(pemData []byte, password string) ([]*x509.Certificate, crypto.P
 		case "CERTIFICATE":
 			cert, err := x509.ParseCertificate(block.Bytes)
 			if err != nil {
-				return nil, nil, fmt.Errorf("block labelled 'CERTIFICATE' could not be pared by x509: %w", err)
+				return nil, nil, fmt.Errorf("block labelled 'CERTIFICATE' could not be parsed by x509: %w", err)
 			}
 			certs = append(certs, cert)
 		case "PRIVATE KEY":
 			if priv != nil {
-				return nil, nil, fmt.Errorf("found multiple blocks labelled 'PRIVATE KEY'")
+				return nil, nil, errors.New("found multiple private key blocks")
 			}
 
 			var err error
-			priv, err = parsePrivateKey(block.Bytes)
+			priv, err = x509.ParsePKCS8PrivateKey(block.Bytes)
+			if err != nil {
+				return nil, nil, fmt.Errorf("could not decode private key: %w", err)
```

Comment:
No, I was just following along with the existing code. I see these `x509` methods call `errors.New`, so I don't see immediate danger in wrapping. But that implies applications can't have taken a dependency on an error type here and it would therefore be safe to switch to `%v` throughout this function. I'll do that.